modules/transformer-batch/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/transformer/batch/Config.scala

```diff
@@ -56,9 +56,34 @@ object Config {
     path: URI,
     compression: Compression,
     region: Region,
-    maxRecordsPerFile: Long
+    maxRecordsPerFile: Long,
+    bad: Output.BadSink
   )
 
+  object Output {
+
+    sealed trait BadSink
+    object BadSink {
+
+      final case class Kinesis(
+        streamName: String,
+        region: Region,
+        recordLimit: Int,
+        byteLimit: Int,
+        backoffPolicy: BackoffPolicy,
+        throttledBackoffPolicy: BackoffPolicy
+      ) extends BadSink
+
+      case object File extends BadSink
```

Comment:
Path is already in `Config.Output`.

If we always use Kinesis, it may be surprising for users as we force them to define Kinesis config, right? So if you're interested in Kinesis output, you define it in config if needed. If not, you can rely on traditional S3 sink and nothing breaks unexpectedly.